src/main/java/core/jdbc/error/code/H2ErrorCode.java

```diff
@@ -0,0 +1,13 @@
+package core.jdbc.error.code;
+
+public class H2ErrorCode extends ErrorCode {
+
+    public H2ErrorCode() {
+        badSqlGrammarCodes = new int[]{42000, 42001, 42101, 42102, 42111, 42112, 42121, 42122, 42132};
+        duplicateKeyCodes = new int[]{23001, 23505};
+        dataIntegrityViolationCodes = new int[]{22001, 22003, 22012, 22018, 22025, 23000, 23002, 23003, 23502, 23503, 23506, 23507, 23513};
+        dataAccessResourceFailureCodes = new int[]{90046, 90100, 90117, 90121, 90126};
+        deadlockLoserCodes = new int[]{50200};
+    }
```

Comment:
부모 클래스의 필드만 사용하는 거라면 상속이 필요 없을 것 같아요.
상수로 선언해도 충분하지 않을까요?
```suggestion
    public static final ErrorCode H2_ERROR_CODE = new ErrorCode();
```